app/_components/Home/Hero/index.jsx

```diff
@@ -1,9 +1,31 @@
-// TODO: Create a Hero component.
-export default function Hero() {
-    return (
-      <main>
-        <h1 className="text-6xl font-bold">Hero</h1>
+// pages/index.js
+import Image from 'next/image';
+import HackathonImage from '../Hero/AIBot.png';
+
+export default function Home() {
+  return (
+    <div className="flex items-center justify-center min-h-screen bg-gray-100 px-4">
```

Comment:
Significant difference between the required UI and the implemented UI.

This can be corrected by removing `min-h-screen` and `bg-gray-100`, and by adding adequate padding and margins.

>Implementation

![localhost_3000_(Laptop 1440)](https://github.com/user-attachments/assets/071d845b-20c5-4d7c-a846-02ea4d78e9dd)

>Requirement

![Screenshot 2024-07-16 at 5 53 53 AM](https://github.com/user-attachments/assets/744a192c-b3e9-40e0-a61b-a8794e8cf9bd)